### PR TITLE
hockeyapp-deploy-android 1.2.0

### PR DESCRIPTION
**Pull request by kdobmayer**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/hockeyapp-deploy-android/1.2.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+  - name: golang
+    bin_name: go
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: .IsCI
+inputs:
+- apk_path: $BITRISE_APK_PATH
+  opts:
+    description: 'The paths are separated with `|` character, eg: `app-armeabi-v7a-debug.apk|app-mips-debug.apk|app-x86-debug.apk`'
```
Comment (koral--) [inline steps/hockeyapp-deploy-android/1.2.0/step.yml]: `app-armeabi-v7a-debug.apk` is not a path but only a filename. Is it really working?

File: steps/hockeyapp-deploy-android/1.2.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+    is_required: true
+    summary: ""
+    title: apk file path
```
Comment (koral--) [inline steps/hockeyapp-deploy-android/1.2.0/step.yml]: Shouldn't it include info that there can be multiple paths? e.g. `APK file path or paths`

File: steps/hockeyapp-deploy-android/1.2.0/step.yml (hunk 1 above)
Comment (viktorbenei) [reply steps/hockeyapp-deploy-android/1.2.0/step.yml]: you're right, unless the apks are relative and in the same dir, but can you please create a Issue / PR on the step's github @koral-- ? This is just the StepLib, the issue should be fixed "upstream" (in the step's repo).

Comment (kdobmayer) [reply steps/hockeyapp-deploy-android/1.2.0/step.yml]: Fixed in [new PR](https://github.com/bitrise-steplib/steps-hockeyapp-android-deploy/pull/12)

**Pull request closed without merging**